docker-compose.yaml

```
@@ -99,7 +77,7 @@ services:
         reservations:
           devices:
             - driver: nvidia
-              device_ids: ["1"]
+              device_ids: ["0"]
```

```suggestion
              device_ids: ["1"]
```
shouldn't we keep it on #1, since that's where cached was? Moving to 0 I think is leading to compute competition with everything else already on 0